k8s/namespaces/rpe/draft-store-service/aat.yaml

```diff
@@ -21,12 +21,13 @@ spec:
         memoryLimits: "3072Mi"
         serviceAccountName: tests-service-account
         keyVaults:
-          "draft-store":
-            resourceGroup: draft-store-service
+          draft-store:
+            excludeEnvironmentSuffix: true
             secrets:
-              s2s-secret-for-tests: S2S_SECRET_FOR_TESTS
-              idam-password-for-tests: IDAM_PASSWORD_FOR_TESTS
-              idam-client-secret-for-tests: IDAM_CLIENT_SECRET_FOR_TESTS
+              - name: s2s-secret-for-tests
+                alias: S2S_SECRET_FOR_TESTS
```

Comment:
There is no reason for giving this i suppose ?
```suggestion
```

Reply:
should be fine I guess as the other 2 secrets are without aliases and its good to test the logic with both the scenarios ?

Reply:
If u want to test, sure lets have one.